product_docs/docs/biganimal/release/getting_started/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Understanding requirements in Azure"
+
+---
+
+By default, Azure sets low resource limits on the number of vCPUs (cores), and Public IP addresses permitted in your subscription. The following sections explain BigAnimal's requirements and recommended resource limits in Azure.
+
+### Public IP addresses limits
+
+A Public IP address is an IP address that used to access your cluster directly over the internet. The number of Public IP addresses must not exceed the limit for a particular region.
+
+##### Default Azure limit
+
+The default Public IP addresses limits for **Basic** and **Standard** type are set to 10. See [Public IP address limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/azure-subscription-service-limits#publicip-address) for more information.
+
+##### Recommended limit
+
+EDB recommends you to increase the Public IP addresses limits for **Basic** and **Standard** type to the number of clusters you plan to deploy in a given region.
+
+### vCPU limits
+
+Any time a new VM is deployed in Azure, the vCPUs for the VMs must not exceed the vCPU limits for the VM size family (see [SKU restrictions](#SKU-restrictions)) or the total regional vCPU limit. Clusters deployed in the region use separate Esv3 virtual machine cores. 
+
+The number of cores depends on the *Instance Type* and *High Availability* options of the clusters. You can calculate the number of Esv3 cores required for your cluster based on the following:
+
+* A Virtual machine instance of type E{N}sv3 uses {N} cores. For example, an instance of type E64sv3 uses 64 Esv3 cores.
+* A cluster running on an E{N}sv3 instance with high availability not enabled uses exactly {N} Esv3 cores.
+* A cluster running on an E{N}Sv3 instance with high availability enabled uses 3 * {N} Esv3 cores.
+
+For example, if you provision the largest virtual machine E64sv3 with high availability enabled, it requires (3 * 64) = 192 Esv3 cores per region. 
+
+BigAnimal uses eight Dv4 virtual machine cores to manage your BigAnimal infrastructure in a particular region. BigAnimal requires an additional eight Dv4 virtual machine cores per region.
```

Comment:
These two sentences are redundant.

Reply:
Removed "BigAnimal uses eight Dv4 virtual machine cores to manage your BigAnimal infrastructure in a particular region."